cookbooks/prescribed_velocity/corner_flow.prm

```diff
@@ -5,6 +5,11 @@ set End time    = 0
 # Load the signal library.
 set Additional shared libraries = ./libprescribed_velocity.so
 
+subsection Discretization
```

Comment:
This is just the cookbook input file, not a test. I am happy to add a test for the DGP case if that would be useful.